src/Symfony/Component/Form/composer.json

```diff
@@ -22,6 +22,7 @@
         "symfony/intl": "^4.4|^5.0",
         "symfony/options-resolver": "^5.1",
         "symfony/polyfill-ctype": "~1.8",
+        "symfony/polyfill-intl-icu": "^1.12",
```

Comment:
just asking in case of typo
you use `symfony/polyfill-intl-icu ^1.21` in the code but here it is `^1.12`, is it desired?

Reply:
good catch! fixed